src/ui/server/controllers/insurance/your-buyer/company-or-organisation/validation/rules/registration-number.ts

```diff
@@ -28,7 +26,7 @@ export const MAXIMUM = Number(FIELDS.COMPANY_OR_ORGANISATION[FIELD_ID].MAXIMUM);
  */
 const registrationNumber = (formBody: RequestBody, errors: object) => {
   if (objectHasProperty(formBody, FIELD_ID)) {
-    return maxLengthValidation(formBody[FIELD_ID], FIELD_ID, ERROR_MESSAGE, errors, MAXIMUM);
+    return maxLengthValidation(formBody[FIELD_ID], FIELD_ID, ERROR_MESSAGE, errors, MAXIMUM_CHARACTERS.BUYER_COMPANY_OR_ORGANISATION_REGISTRATION_NUMBER);
```

Comment:
dereference?

Reply:
Replaced with the following - I don't think we should dereference in such validation functions, so it's very clear it's a "max characters" definition

```js
BUYER.COMPANY_OR_ORGANISATION
BUYER.REGISTRATION_NUMBER
```